src/System.ComponentModel.Primitives/src/System/ComponentModel/ComponentModelExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.ComponentModel
+{
+    public interface IIsDefaultAttribute
```

Comment:
Yes - this is a stopgap for now and is more of an internal implementation to unblock us. I have a design for a mapping of known default attributes that can remove the dependency on this API. I haven't merged it in yet, as I want to wait until the primitives library has the surface area we need. See https://github.com/twsouthwick/corefx/commit/ba92b5475f0356dda63e71eb0f0c4721540a82a2 for the fix.